pkg/commands/container.go

```diff
@@ -248,7 +248,11 @@ type ContainerCliStat struct {
 
 // GetDisplayStrings returns the dispaly string of Container
 func (c *Container) GetDisplayStrings(isFocused bool) []string {
-	return []string{c.GetDisplayStatus(), c.Name, c.GetDisplayCPUPerc()}
+	image := c.Container.Image
+	if strings.HasPrefix(image, "sha256:") {
+		image = image[7:20]
+	}
+	return []string{c.GetDisplayStatus(), c.Name, c.GetDisplayCPUPerc(), image}
```

Comment:
```suggestion
	return []string{c.GetDisplayStatus(), c.Name, c.GetDisplayCPUPerc(), utils.ColoredString(image, color.FgMagenta)}
```
What are your thoughts on giving this a color other than the default? I think that makes the CPU percentage easier to read:
![image](https://user-images.githubusercontent.com/8456633/62534053-800fe100-b88b-11e9-87d5-120fdc990950.png)